ietf/templates/submit/announce_to_authors.txt

```diff
@@ -1,24 +1,23 @@
-{% autoescape off %}
-A new version of I-D, {{ submission.name }}-{{ submission.rev }}.txt
-has been successfully submitted by {{ submission.submitter_parsed.name }} and posted to the
-IETF repository.
+{% autoescape off %}{% filter wordwrap:78 %}A new version of Internet-Draft {{ submission.name }}-{{ submission.rev }}.txt has been successfully submitted by {{ submission.submitter_parsed.name }} and posted to the
+IETF repository.{% endfilter %}
 
-Name:		{{ submission.name }}
-Revision:	{{ submission.rev }}
-Title:		{{ submission.title }}
-Document date:	{{ submission.document_date|date:"Y-m-d" }}
-Group:		{{ group }}
-Pages:		{{ submission.pages }}
-URL:            {{ settings.IETF_ID_ARCHIVE_URL }}{{ submission.name }}-{{ submission.rev }}.txt
-Status:         {{ settings.IDTRACKER_BASE_URL }}{% url 'ietf.doc.views_doc.document_main' name=submission.name %}{% if submission.xml_version == "3" %}
-Html:           {{ settings.IETF_ID_ARCHIVE_URL }}{{ submission.name }}-{{ submission.rev }}.html{% endif %}
-Htmlized:       {{ settings.IDTRACKER_BASE_URL }}{% url 'ietf.doc.views_doc.document_html' name=submission.name %}
-{% if submission.rev != "00" %}Diff:           {{rfcdiff_base_url}}?url2={{ submission.name }}-{{ submission.rev }}{% endif %}
+Name:     {{ submission.name }}
+Revision: {{ submission.rev }}
+Title:    {{ submission.title }}
+Date:     {{ submission.document_date|date:"Y-m-d" }}
+Group:    {{ group }}
+Pages:    {{ submission.pages }}
+URL:      {{ settings.IETF_ID_ARCHIVE_URL }}{{ submission.name }}-{{ submission.rev }}.txt
+Status:   {{ settings.IDTRACKER_BASE_URL }}{% url 'ietf.doc.views_doc.document_main' name=submission.name %}{% if submission.xml_version == "3" %}
+HTML:     {{ settings.IETF_ID_ARCHIVE_URL }}{{ submission.name }}-{{ submission.rev }}.html{% endif %}
+HTMLized: {{ settings.IDTRACKER_BASE_URL }}{% url 'ietf.doc.views_doc.document_html' name=submission.name %}
```

Comment:
What is sthe difference between `HTML` and `HTMLized`? Can we find a better word to make that more obvious?

Reply:
For modern documents (anything submitted in xmlrfc), HTML is the html rendering from xml2rfc. HTMLized is the representation of the text rendering as html with some heursitic links created. We can HTMLize documents that are submitted only as text. We cannot make v3 HTML them.

What the datatracker /doc/html page shows _looks_ (mostly) like the htmlized version, whether it eats .txt and htmlizes it or uses css to format the v3 HTML to look like the HTMLized.

And yes, this is not easy to explain.

Reply:
Is it time to just drop the HTMLized?
Can we add it only if txt was submitted?

I guess the bigger question is when would we want to provide both?

Reply:
I think we should do a pass over all email announcements and make sure we link to the metadata page in the datatracker in most cases, rather than including per-rendering links (or using names baking in renderings like `.txt`).

Reply:
> Is it time to just drop the HTMLized

No. Your question isn't really just about "in the announcement of new Internet-Draft versions" its a thing that we have to deal with in general, including the pre-v3 RFCs. The /doc/html pages _use_ that format so that we can provide the same kind of view for older and newer documents. Unless we somehow convinced the community that the v3 html rendering's CSS change to be what is produced at /doc/html, we need two terms for the two ways of looking at things.

This conversation should move out of the ticket - probably onto tools-discuss@